sdks/go/pkg/beam/runners/prism/internal/execute.go

```diff
@@ -70,6 +71,13 @@ func RunPipeline(j *jobservices.Job) {
 		j.Failed(err)
 		return
 	}
```

Comment:
more of an understanding question: in case of a `Cancel`, the err here will be nil which is returned from [here](https://github.com/apache/beam/pull/30178/files#diff-a9df335416772bd68225ef5d43f089bd6f0eb678f5472330168051d9bec3df1eR282)?

Reply:
I think so yes. More importantly, I reminded myself to find where the test is that asserts this behavior.

Reply:
@damondouglas TODO

Reply:
okay thanks!